spec/cloudflair/api/zone/settings/development_mode_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'spec_helper'
 
 describe Cloudflair::DevelopmentMode do
+  subject(:development_mode) { Cloudflair.zone(zone_identifier).settings.public_send setting_identifier }
```

Comment:
can you infer the subject name from `setting_identifier`? Else I think it should not have a name at all, i.e. be like this: `subject { Cloudflair.zone .... }`. This is because I want to extract the common parts out of each of those specs any time soon.